src/components/Player/Player.tsx

```diff
@@ -17,20 +17,18 @@ export interface IPlayerProps {
 }
 
 export default function Player(props: IPlayerProps): JSX.Element {
-  const { t } = useTranslation();
+  // const { t } = useTranslation();
```

Comment:
@francistse23 we can remove this import if it's not needed.

Reply:
i think we were using translation for the input placeholder.

since chilblane changed `player` to `${color}`, we might need to add that back in once we set translations straight

Reply:
Got it.